spec/models/info_request_spec.rb

```diff
@@ -1209,6 +1209,15 @@
           with_message( "Unknown key 'random' for '#{info_request.law_used}'")
       end
 
+      context 'using the non default language' do
```

Comment:
I actually don't think we should test for this.

I'd _expect_ this to just work. I wouldn't write a spec each time we added a translated string, so I don't think we should here.

A better thing to do would be to add a custom hound check to assert that we haven't assigned translated values to constants.